org.jacoco.core/src/org/jacoco/core/internal/instr/ProbeInserter.java

```diff
@@ -62,7 +62,7 @@
 		for (final Type t : Type.getArgumentTypes(desc)) {
 			pos += t.getSize();
 		}
-		variable = pos;
+		variable = pos + 1;
```

Comment:
I would add some documentation about the "magic" +1. Either here and maybe in addition in the class documentation to explain the general approach.